server/@types/models/index.d.ts

```diff
@@ -43,6 +43,7 @@ declare module 'models' {
     sentence?: Sentence
     terrorRelated?: boolean
     legacyData: Record<string, never>
+    fineAmount?: number
```

Comment:
A fineAmount is associated to a Sentence not an Offence, can this be moved to the Sentence object below

Reply:
A sentence can't have multiple offences, one offence to one sentence. The fine amount comes after selecting a Imprisonment in default of a fine sentence type not outcome. The sentence type is associated to the sentence not the offence so therefore the fine amount if associated to the sentence not the offence

Reply:
I've adjusted this